e2e/pages/citizen/createCase/C100/casePartyDetails/applicantAddressSelectPage.ts

```diff
@@ -96,7 +96,7 @@ export class ApplicantAddressSelectPage {
       ),
       Helpers.checkVisibleAndPresent(
         page,
-        `${Selectors.GovukErrorMessage}:text-is("${ApplicantAddressSelectContent.errorMessage}")`,
+        `${Selectors.GovukErrorMessageCitizen}:text-is("${ApplicantAddressSelectContent.errorMessage}")`,
```

Comment:
I think this is a duplicate selector. There's one called ErrorMessage which is also `.govuk-error-message`